### PR TITLE
[Pages] Pelican framework guide

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/pages/framework-guides/deploy-a-pelican-site.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+# Deploy a Pelican site
+
+[Pelican](https://docs.getpelican.com) is a static site generator, written in Python. With Pelican, you can write your content directly with your editor of choice in reStructuredText or Markdown formats.
+
+## Create a Pelican project
+
+To begin, create a Pelican project directory. `cd` into your new directory and run:
+
+
```
Comment (WalshyDev) [inline content/pages/framework-guides/deploy-a-pelican-site.md]: double new line
```suggestion
```

**Review by WalshyDev — APPROVED** [review]
LGTM

File: content/pages/framework-guides/deploy-a-pelican-site.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+$ python3 -m pip install pelican
+```
+
+Then run:
+
+```sh
+$ pip freeze > requirements.txt
+```
+
+Create a directory in your project named `content`:
```
Comment (marciocloudflare) [inline content/pages/framework-guides/deploy-a-pelican-site.md:24]: Should these be numbered steps?

**Review by marciocloudflare — APPROVED** [review]

File: content/pages/framework-guides/deploy-a-pelican-site.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```sh
+$ mkdir content
+```
+
+This is the folder name that you will set in the build command.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-pelican-site.md]:
```suggestion
This is the directory name that you will set in the build command.
```

**Pull request merged**